### ISSUE TITLE
Resolve imported classes for phpdoc blocks

### ISSUE THREAD

**Issue by asgrim** [Roave/BetterReflection#5]
Reported initially in Roave/StrictPhp#27

We can move this functionality from [Sevenificator](https://github.com/GeeH/Sevenificator) into Better Reflection, meaning that both Sevenificator and StrictPhp won't need to maintain separate implementations

e.g. 
```php
namespace RstGroup;

use Doctrine\DBAL\Logging\DebugStack;
use Psr\Log\LoggerInterface;

class SqlLogger extends DebugStack
{
    /**
     * @var LoggerInterface
     */
    protected $logger;

    public function __construct(LoggerInterface $logger)
    {
        $this->logger = $logger;
    }
}
```

The reflection should auto-resolve `LoggerInterface` to the imported `Psr\Log\LoggerInterface`. If the class cannot be resolved... throw exception maybe?

**Comment by asgrim** [comment]
See fix from Roave/StrictPhp#29 ... this will need to be implemented here too

**Comment by Ocramius** [comment]
Handled in #52
